docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -566,6 +566,33 @@ Here one could find description of general HTTP properties applied across all th
 
 |===
 
+== Properties externalization
+
+. Declare https://en.wikipedia.org/wiki/Environment_variable[environment variable] containing desired property value and then 
+point the property to the declared environment variable:
++
+[source,properties]
+----
+bdd.variables.global.input-text=${MY_INPUT_TEXT}
+----
++
+[source,shell]
+----
+export MY_INPUT_TEXT=kekw
+./gradlew runStories
+----
+
+. Pass the property with its value into a test run using command line arguments prepending it with the `-Pvividus.` prefix. 
```

Comment:
we need to indicate that this approach is valid for execution via Gradle only

Reply:
isn't it reflected in the example below?

Reply:
it's not clear, because users may think it could be used for the embedded sh-script as well, but it's Gradle-only